manifests/ownership.pp

```diff
@@ -1,19 +1,25 @@
-# Enabling this class will take ownership of the TPM in the system,
-# using an auto-generated password created with simplib's passgen.
+# @summary Take ownership of the TPM in the system, using an auto-generated password created with simplib's passgen.
 #
 # The password must be generated with passgen in order for most of the facts
 # to be functional post-ownership, as the tpm commands from tpm-tools
 # require the owner password.
 #
-# @param owner_pass The TPM owner password
+# @param owned
+#   Whether or not the module should take ownership
 #
-# @param srk_pass The TPM SRK password. This is defaulted to an empty
-#   because according to the [trousers documentation](http://trousers.sourceforge.net/pkcs11.html)
-#   it needs to be null to be useful.
+# @param owner_pass
+#   The TPM owner password
 #
-# @param advanced_facts This option will enable facts that require
-#   the owner password to function. The password will be on the client
-#   filesystem (in `$vardir/simp`) if enabled.
+# @param srk_pass
```

Comment:
also in lib/puppet/type/tpm_ownership.rb we use srk**-**pass instead of srk_pass - that needs to be fixed